django/contrib/messages/test.py

```diff
@@ -0,0 +1,21 @@
+from .api import get_messages
+from .constants import DEFAULT_LEVELS
+
+
+class MessagesTestMixin:
+    def assertMessages(self, response, expected_messages):
+        level_to_name = {label: level for level, label in DEFAULT_LEVELS.items()}
```

Comment:
Should not we use `get_level_tags()` instead of `DEFAULT_LEVELS`?

Reply:
It definitely should. Thanks!